build.bat

```diff
@@ -0,0 +1,51 @@
+@echo off
+MKDIR app src py-spy 2>NUL
+REM Download Build Tools for Visual Studio
+REM https://aka.ms/vs/17/release/vs_BuildTools.exe
+WHERE python
+IF %ERRORLEVEL% NEQ 0 (
+	ECHO python wasn't found. Attempting to install...
+	ECHO Shell will close once complete. Re-run build.bat to proceed
+	CALL .\dep\python-3.9.13-amd64.exe /passive PrependPath=1 Include_test=0
+)
+REM Get Python version
+FOR /f "tokens=1-2" %%i in ('python --version') do (
+	set PYTHON_VERSION=%%j
+	IF /i "%PYTHON_VERSION:~0,1%" == "3" (
+		ECHO Python version is valid
+	) ELSE (
+		ECHO Python 3 is required. Attempying to install...
+		ECHO Shell will close once complete. Re-run build.bat to proceed
+		.\deb\python-3.9.13-amd64.exe /passive PrependPath=1 Include_test=0
+	)
+)
+@echo Installed python version: %PYTHON_VERSION%
+IF EXIST .\py-spy\py-spy.exe (
+	ECHO Found py-spy.exe
+) ELSE (
+	ECHO Building py-spy executable...
+	CALL build-pyspy.bat
+)
+CD app
+
+WHERE pip
+IF %ERRORLEVEL% NEQ 0 (
+	ECHO pip wasn't found.
+	ECHO Install pip for current and re-run build.bat
+	EXIT /B -1
+)
+Rem pip install --no-cache-dir --user --upgrade pip
+MKDIR granulate-utils 2> NUL
```

Comment:
What are you doing in the next 10-15 lines? Files are copied around and you re-create `granulate-utils`, what's the need for it? Can't we work in the existing source directory?

Reply:
I had to re-create granulate-utils because the requirements.txt specifies a relative path for granulate-utils as follows. If it isn't at that location it fails. Also, I tried to put all build artifacts in the app directory so as to leave the source directory untouched. Doing so, doesn't permit me to use the relative path defined below in the requirements.txt file without making a copy of granulate-utils. I hope that makes sense

./granulate-utils/

Reply:
I see. I don't think it matters much if the source directory remains untouched - after all, it's just a build machine which runs it, so I think we can do the same as done in the Linux build scripts (i.e install everything in-place). Up to you

Reply:
I'll keep it the way it is for now as it helps to be able to clean up between runs.